pkg/operator/endpoints/errors.go

```diff
@@ -108,3 +120,38 @@ func ErrorAuthForbidden() error {
 		message: "invalid AWS credentials; run `cortex configure` to configure your CLI with credentials for any IAM user in the same AWS account as the operator",
 	}
 }
+
+func ErrorAppNotDeployed(appName string) error {
+	return Error{
+		Kind:    ErrAuthForbidden,
+		message: fmt.Sprintf("app %s is not deployed", s.UserStr(appName)),
+	}
+}
+
+func ErrorFormFileMustBeProvided(fileName string) error {
+	return Error{
+		Kind:    ErrFormFileMustBeProvided,
+		message: fmt.Sprintf("request form file %s must be provided", s.UserStr(fileName)),
+	}
+}
+
+func ErrorQueryParamRequired(paramNames ...string) error {
```

Comment:
I think this should just take one string, otherwise it could be confusing how they are joined (e.g. with "or" or "and")